### PR TITLE
Fake the install status

### LINKED ISSUE #312: Fake the installation and uninstallation of add-ons
When we have the state mapped out in #311 we can add some fake handlers so that clicking the button "installs" or "uninstalls" an add-on until we get #308 figured out.

### PR DESCRIPTION
**Pull request by mstriemer**
disco-fake-spinner-two mov

This doesn't actually install anything it just runs the switch through the install and uninstall steps.

Fixes #312.

### CODE REVIEW

File: src/disco/containers/InstallButton.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class InstallButton extends React.Component {
   render() {
     const { status, downloadProgress } = this.props;
 
-    if (validStates.indexOf(status) === -1) {
+    if (!validStates.includes(state)) {
```
Comment (muffinresearch) [inline src/disco/containers/InstallButton.js]: Should be status :)

File: src/disco/containers/InstallButton.js [hunk 2]
```diff
@@ -7,40 +7,52 @@ import 'disco/css/InstallButton.scss';
...
+    }
   }
 
   render() {
-    const { state, progress } = this.props;
+    const { status, downloadProgress } = this.props;
 
-    if (validStates.indexOf(state) === -1) {
-      throw new Error('Invalid add-on state');
+    if (!validStates.includes(state)) {
```
Comment (muffinresearch) [inline src/disco/containers/InstallButton.js]: Should be status :)

**Comment by mstriemer** [comment]
Not quite sure how I managed that. Fixed now.

**Comment by muffinresearch** [comment]
r+

**Pull request merged**